components/carousel/utils/useCarouselEvents.ts

```diff
@@ -0,0 +1,151 @@
+import type { CarouselNFT } from '@/components/base/types'
+import type { NFTWithMetadata } from '@/composables/useNft'
+import { isBeta, isProduction } from '@/utils/chain'
+import { formatNFT } from '@/utils/carousel'
+
+import latestEvents from '@/queries/subsquid/general/latestEvents.graphql'
+import latestEventsRmrkv2 from '@/queries/subsquid/ksm/latestEvents.graphql'
+
+interface Types {
+  type: 'latestSales' | 'newestList'
+}
+
+const limit = isProduction ? 15 : 8
+
+const nftEventVariables = {
+  latestSales: { interaction_eq: 'BUY' },
+  newestList: { interaction_eq: 'LIST' },
```

Comment:
missing `price_not_eq: '0'`?

Reply:
oh correct, added in here [`cea0efe` (#6863)](https://github.com/kodadot/nft-gallery/pull/6863/commits/cea0efe91bf7e7de28e6362f04840833f237a452)